ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/animation/TextViewAnimation.kt

```diff
@@ -0,0 +1,22 @@
+package org.cru.godtools.tutorial.animation
+
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
+import androidx.annotation.StringRes
+import androidx.appcompat.widget.AppCompatTextView
+
+fun AppCompatTextView.animateToNextText(@StringRes animateNextText: Int) {
+    alpha = 1f
+    animate().setDuration(3000).alpha(0f).setListener(object : AnimatorListenerAdapter() {
+        override fun onAnimationEnd(animation: Animator?) {
+            super.onAnimationEnd(animation)
+            fadeInToNewText(animateNextText)
+        }
```

Comment:
a 3 second fade out is pretty slow, you will probably want it to be on the order of 500-1000 ms

Reply:
same thing with the fade in